src/PgManyToManyRelationPlugin.js

```diff
@@ -290,7 +290,8 @@ module.exports = function PgManyToManyRelationPlugin(
                                     leftKeyAttribute.name
                                   )})`
                                 );
-                              }
+                              },
+                              queryBuilder.context
```

Comment:
For full compatibility, please pass rootValue also:

```suggestion
                              queryBuilder.context,
                              queryBuilder.rootValue
```